docs/guide/introduction.md

```diff
@@ -97,4 +97,96 @@ The rest of the documentation will include many specific, practical examples of
+
+Our blog post module's schema might look something like this:
+
+```javascript
+// modules/blog-post/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  fields: {
+    add: {
+      authorName: {
+        label: 'Author name',
+        type: 'string'
+      },
+      body: {
+        label: 'Blog post body',
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {}
+          }
+        }
+      }
+    },
+    group: {
+      blogFields: [ 'authorName', 'body' ]
+    }
+  }
+};
+```
+
+Content schemas are configured in the `fields` setting. In this case, `fields` has two subsections: `add`, where fields are added to the schema, and `group`, which organizes the fields for the user interface.
+
+Each property in the `add` object is a field you are including in the schema. Each property in `group` is a section of the interface, set to an array of fields to include in that section.
+
+See the reference documentation on [the `fields` setting](/reference/module-api/module-overview.md#fields) and [individual field types](/reference/field-types/) for more information.
+
+### Using existing field groups
+
+Fields that a piece type inherits will likely already be in field groups. This is certainly true for the default fields, `title`, `slug`, and `visibility`. You can add new fields into these groups and rearrange them if needed. There are a few things to keep in mind as you do.
+
+**It's fairly simple to see what the existing groups are.** Working with inherited fields and field groups is harder when you don't know what they are. You can log them in your terminal easily from the new module's [initialization function](/reference/module-api/module-overview.md#initialization-function).
+
+```javascript
+// modules/product/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  init (self) {
+    console.log(self.fieldsGroups);
+
+    // Output:
+    // {
+    //   basics: { label: 'Basics', fields: [ 'title' ] },
+    //   utility: { fields: [ 'slug' ] },
+    //   permissions: { label: 'Permissions', fields: [ 'visibility' ], last: true }
+    // }
+  }
+};
+```
+
+The `init` function runs once on start up and has access to the module as an argument. By the time it runs, the field groups have been compiled into an object named `fieldsGroups`. If you haven't added any fields yet you can log this to see what you are working with.
+
+::: note
+You will see a `trash` field in the log output. The interface does not show this as a normal field, but
+:::
+
+**If you name an existing field group in your configuration, the fields in that group will be ungrouped.** For example, `title` is in the default "Basics" group. If you add a `basics` group in your field configuration and do not include `title` in its `fields` array, it will no longer be in any group. Fields that are not part of any group will appear in an "Ungrouped" tab in the interface.
```

Comment:
Yes, that's correct. It's annoying. I wish there was a better way.

Reply:
@abea Yeah.  I definitely found myself thinking "why is it this way" as I was reading. For now though:

```suggestion
**Using an existing group name in your configuration will cause fields already in that group to become "ungrouped"**. For example, `title` is in the default "Basics" group. If you add a `basics` group in your field configuration and do not include `title` in its `fields` array, it will no longer be in any group. Fields that are not part of any group will appear in an "Ungrouped" tab in the interface.
```